CONTRIBUTING.md

```diff
@@ -139,6 +143,83 @@ The description field should
 - be neutral and not use biased additions like ~~leading~~, ~~innovative~~, ~~flexible~~ or buzzwords (~~performance~~, ~~next generation~~)
 - not contain hyperlinks or markup
 
+
+### Compatibility information
+
+FMU exporting and importing tools provide information to their users on how they have validated their FMI support.
+Providing and maintaining this information is in the responsibility of the tool vendors. On the tools page of the FMI Webpage, a link to a repository or webpage of the tool vendor will be provided. Tool vendors are encouraged to clearly mark this webpage as provided by them and not by the Modelica Association or the FMI Project, e.g., by prominently placing their logo on this page.
+
+#### Exporting tools
+
+* List the importing tools (including version information), with which the FMUs from this exporter have been imported and simulated successfully.
+This could be (free or commercial) tools available to the tool vendor of the exporting tool. Or the tests could be performed by third parties (users, tool vendors of importing tools). 
+  * optionally additional information needed to reproduce the results of the importers (e.g., compiler flags)
+* Encouragement to provide example FMUs and any information needed to run this FMU and check expected results
+  * check with one of the available checkers before uploading the FMUs (fmpy, VDM Checker, FMU Compliance Checker for FMI 2.0) --> self-certification
+  * description, including with which version of the tool the FMU was generated
+  * (copy description of files from https://github.com/modelica/fmi-cross-check/blob/master/FMI-CROSS-CHECK-RULES.md#914-files-to-submit: TODO: Define if reference result refers to result of the simulation of the original model in the exporting tool, or to the simulation of the FMU)
+  * recommendation: structure your repository according to FMI versions, FMI interface types (ME, CS, SE), platforms (consider to provide source code FMUs)
```

Comment:
> Should this be a recommendation? Are there good reasons not to enforce this? by enforcing it it is possible to script import which will be necessary if one wants to test all FMU's before release of new tool versions (which I used to do with the old cross check). If it is not enforce and we end up with many different tools not flowing this it would be a shame since I don't know would would want to do that check manually.

I personally agree. But as the repo is under control of the tool vendor, it will be hard to enforce anyway.
@t-sommer : could you please comment? You suggested to keep this more flexible.

Reply:
@t-sommer : do you have a proposal to avoid duplication of FMUs for multiple FMU kinds, platforms? Would it be just set folders without a substructure for FMU kind or platforms?
```
2.0
..
3.0
    BouncingBall
    ...
    CoupledClutches
      CoupledClutches.fmu 
      CoupledClutches_in.csv 
      CoupledClutches_ref.csv 
      CoupledClutches_ref.opt
      Readme.txt
```

The information on supported platforms and FMU kinds is already present in the modelDescription.xml, so we do not have to duplicate it. All or the different variants would share the same reference solution. (if this does not fit, one could generate multiple folders)

Could you also provide a hint on sample tooling (fmpy?) how to process them?

Reply:
The folder structure can be chosen by the vendor to match the capabilities of the tool, e.g. by tool version. As you say, there is no need for a fixed structure, because the info can be extracted from the FMUs themselves.

Reply:
@t-sommer , @KarlWernersson : I comitted a proposal according to Torsten's suggestion. 
Is this OK for you?